extensions/wikia/AdEngine/js/DOMElementTweaker.js

```diff
@@ -0,0 +1,70 @@
+/*global define*/
+define('ext.wikia.adEngine.domElementTweaker', [
+	'wikia.log',
+	'wikia.window'
+], function (log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.domElementTweaker',
+		rclass = /[\t\r\n]/g;
+
+	function removeClass(element, cls) {
```

Comment:
I wonder if it's worth to add unit tests to this method, it doesn't have to be right now, however the `while` loop scares me a little.